ansible/roles/test/templates/bgp_speaker_route.j2

```diff
@@ -0,0 +1,8 @@
+{% if addr_family == 'ipv6' %}
+{{announce_prefix}} [{% for member in minigraph_portchannels['PortChannel01'].members %}{{ '%d' % minigraph_port_indices[member]}}{% if not loop.last %} {% endif %}{% endfor %}]{% if not loop.last %} {% endif %}
```

Comment:
Can we avoid hard code "PortChannel01" here? It will fail easily if the name of the port channel changes.

Reply:
I investigated it and looks like we cannot avoid hard code if we want to use only one specific port-channel.
In case of using all port-channels (like FIB test does) we can just iterate through the all LAGs and so avoid hard code.
But, as we discussed earlier, we should use only one port-channel as a next-hop to avoid ECMP.

Reply:
Shall we consider use a j2 template to select one ipv6 portchannel, and keep track of its name to avoid hardcoding in both places?